<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartForm</class>
 <widget class="QWidget" name="ChartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1700</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>14</number>
     </property>
     <item>
      <widget class="QCheckBox" name="avgBox">
       <property name="text">
        <string>平均值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="maxBox">
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chargeBox">
       <property name="text">
        <string>放电频次</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maxButton">
       <property name="text">
        <string>放大</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minButton">
       <property name="text">
        <string>缩小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>拖动</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reportButton">
       <property name="text">
        <string>报表</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QChartView" name="chartView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
